biofefi/feature_importance/call_methods.py

```diff
@@ -101,9 +98,12 @@ def save_importance_results(
             rotation=plot_opt.angle_rotate_yaxis_labels,
             family=plot_opt.plot_font_family,
         )
-        ax.set_xlabel(ax.get_xlabel(), family=plot_opt.plot_font_family)
-        ax.set_ylabel(ax.get_ylabel(), family=plot_opt.plot_font_family)
-        ax.set_title(ax.get_title(), family=plot_opt.plot_font_family)
+        ax.set_xlabel("Feature", family=plot_opt.plot_font_family)
+        ax.set_ylabel("Importance", family=plot_opt.plot_font_family)
+        ax.set_title(
+            f"{feature_importance_type} - {model_type}",
+            family=plot_opt.plot_font_family,
+        )
         fig.savefig(save_dir / f"{model_type}-bar.png")
 
         if feature_importance_type == "SHAP":
```

Comment:
Can we use an enum instead of the magic string "SHAP"?

Reply:
I agree. However, I think this can come in a different PR is there is a little bit of complexity around the feature importance types that I need to have a think how to separate them out. That is out of scope for this PR which is about plot styling

Reply:
Alright then.